### PR TITLE
Add Apple Pay Overview

### PR DESCRIPTION
**Pull request by djejaquino**
## Description

Adds Apple Pay Overview page

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: docs/guides/apple-pay/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+In this page, we will explore Apple Pay concepts and take a deep dive on how it works. If you are looking for hands-on guides, check out the [Merchant Setup](/docs/guides/apple-pay/setup) and [Accept Payments](/docs/guides/apple-pay/accept) pages we prepared for you.
+
+## Glossary
+
+- **FPAN**: Funding Primary Account Number. This is the physical or virtual card number, or just "PAN".
+- **DPAN**: Device Primary Account Number. Apple's equivalent to a Network Token. Also known as  "DAN" or "Application PAN".
+- **BIN**: Bank Identifier Number. The first 8 digits of a card number. These are usually preserved when generating DPANs from FPANs.
+- **Apple Pay Token**: [payment token object](https://developer.apple.com/documentation/passkit/apple_pay/payment_token_format_reference) issued by the device upon user approval via biometric authentication (Face ID or Touch ID). It carries transaction information, including an encrypted form of the DPAN and transaction cryptogram.
+- **Issuer**: bank or bank-sponsored financial institution that issues the payment card to the cardholder and is responsible for approving transactions
+- **Acquirer**: bank or bank-sponsored entity that will
```
Comment (greathouse) [inline docs/guides/apple-pay/overview.mdx:16]: **Acquirer** definition is incomplete.

File: docs/guides/apple-pay/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+- **TSP**: Token Service Provider are entities registered with EMVCo to generate unique replacement values for PANs, aka "tokens". In many cases, these are the Card Networks.
+- **Secure Element**: Industry-standard certified chip present in Apple hardware, such as iPhones, iPads, Apple Watch, etc.
+
+## Adding Card to Apple Wallet
+
+Adding a card to Apple Wallet can either be done by manually adding card details using Apple Wallet UI or using apps that can automatically add cards to the user's wallet - also known as push-provisioning.
+
+Regardless of the method, when a user adds a card to Apple Pay, many systems are involved into ensuring that the sensitive cardholder data is protected and ready to use when it is time to process transactions.
```
Comment (greathouse) [inline docs/guides/apple-pay/overview.mdx:24]: "many systems are involved into ensuring that the sensitive cardholder data"
Should that be "many systems are involved ~~into~~ *in* ensuring that the sensitive cardholder data"? I don't know, "into" sounds incorrect to me.

File: docs/guides/apple-pay/overview.mdx (hunk 1 above)
Comment (djejaquino) [inline docs/guides/apple-pay/overview.mdx:16]:
```suggestion
- **Acquirer**: bank or bank-sponsored entity that is responsible for processing credit or debit card transactions on behalf of merchants, facilitating the transfer of funds from the customer's account to the merchant's account.
```

**Review by greathouse — APPROVED** [review]

**Pull request merged**